addons/cookoff/functions/fnc_cookOff.sqf

```diff
@@ -113,11 +113,20 @@ if (local _vehicle) then {
 
         [_vehicle, _fnc_FlameEffect, 12] call _fnc_FlameEffect; // recursive function
 
+        private _randomPosition = _vehicle getPos [100, random 360];
+
         {
-            if (local _x && {!(_x call EFUNC(common,isPlayer))}) then {
-                _x action ["Eject", _vehicle];
+            _x leaveVehicle _vehicle;
+            
+            if (leader _x == _x) then {
+                private _wp = group _x addWaypoint [_randomPosition, 0, 1];
```

Comment:
Sorry, Silent. I'm bad with faces, names and apparently online aliases too it seems. :|